src/App.tsx

```diff
@@ -1,11 +1,51 @@
 import React from "react"
-
+import "./styles.scss"
 import Navbar from "./components/Navbar/index"
 import { ReactComponent as Logo } from "./assets/profile.svg"
+import CharacterList from "./components/CharacterList"
 function App() {
+  interface ICharacter {
+    name: string
+    id: number
+    gender: string
+    spacies: string
+    origin: string
+    type: string
+    image: string
+    location: string
+    episodes: object[]
+  }
```

Comment:
Don't define interface or type inside the components. If you want to define them in the same file, define them at the top before the component definition. Also, you can define them in a separate type file and then export and use it wherever you want.